libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/FlagEntitlementType.java

```diff
@@ -10,5 +10,6 @@
 package org.elasticsearch.entitlement.runtime.policy;
 
 public enum FlagEntitlementType {
-    SYSTEM_EXIT;
+    SYSTEM_EXIT,
```

Comment:
I know @rjernst suggested that these might all be separate entitlement checks. It does seem like a lot of boiletplate, though, to add a check for each boolean entitlement.

Reply:
I want to keep the methods in PolicyManager to a minimum, but yes, proabably we'll have like 10s of those?
It's easy to change though, so I'll let it as-is and we can revisit it at any time.